File: src/2006/4021277323/main.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    int a; cin>>a;
    int b; cin>>b;
    int max = (a > b) ? a : b;
    int min = (a > b) ? b : a;
    int i;
    for (i = min ; i <= max ; i++)
    {
        bool flg = true;
        for (int j = 2; j < i ; j++)
        {
            if (i%j == 0)
            {
                flg = false;
            } 
        }

        if (flg && i!=1)
        {
            cout<< i <<endl;
        }
        
    }
    


    return 0;
}

File: src/2006/4021277323/main.py

a = int(input())
b = int(input())
max = a if a > b else b
min = b if a > b else a

for i in range(min, max+1):
    flg = True
    for j in range(2, i):
        if i % j == 0:
            flg = False
    if flg and i != 1:
        print(i)
